src/fetch.js

```diff
@@ -79,7 +82,16 @@ function extract(data) {
         // keywords: stepsToTagScores(st)
     }]
 }
+function parseStep({ name, text }) {
+	// NOTE: some of the steps have name; if not, we use the first sentence
+	return name ? {
+		description: name, text
```

Comment:
multiple lines are easier to read
```suggestion
		description: name, 
		text
```